libs/blocks/marquee/mnemonic-list.js

```diff
@@ -0,0 +1,26 @@
+import { getConfig, createTag, loadStyle } from '../../utils/utils.js';
+
+export const decorateMnemonicList = (container) => {
+  if (!container) return;
```

Comment:
this check can be done in marquee.js even before loading this file.

Reply:
Fixed